book/src/first-party-code.md

```diff
@@ -22,10 +22,21 @@ and requires the user to specify the intended behavior. Specifically, if there
 exists a public crate with the same name and version as a given first-party
 crate, cargo-vet will require a policy entry for that crate specifying
 `audit-as-crates-io` as either true or false. If it's set to true, cargo-vet
-will perform audit enforcement as if the crates.io version were being used.
+will perform audit enforcement.
 
-It's important to note that any audits for such derived crates still correspond
-to the crates.io version. This is what `inspect` and `certify` will display, and
-this is what you should review before certifying, since others in the ecosystem
-may rely on your audits when using the original crate without your particular
-modifications.
+When enabled for a git dependency, this enforcement is precise. It requires an
+audit for the base published version that exists on crates.io, and then one or
```

Comment:
It currently doesn't actually require an audit for the "base published version" (as we discussed, the git dependency might have an unpublished version or a version which is not as related to the actual version) - it's just acting like a normal revision.

This wording perhaps should be updated to instead suggest that things "should" be audited as deltas from the base version?